src/app/modules/main/module.nim

```diff
@@ -189,7 +192,7 @@ method load*[T](
   let (unviewedCount, mentionsCount) = self.controller.getNumOfNotificaitonsForChat()
   let hasNotification = unviewedCount > 0 or mentionsCount > 0
   let notificationsCount = mentionsCount
-  let chatSectionItem = initItem(conf.CHAT_SECTION_ID, SectionType.Chat, conf.CHAT_SECTION_NAME, "", "",
+  let chatSectionItem = initItem(conf.CHAT_SECTION_ID, SectionType.Chat, conf.CHAT_SECTION_NAME, false, "", "",
```

Comment:
Suggestion: for all those empty or boolean values, it's nice to have the property name, like:
```nim
initItem(conf.CHAT_SECTION_ID, SectionType.Chat, conf.CHAT_SECTION_NAME, isAdmin = false, name = "", whatever = "",
```
disclaimer: I don't know the names of the properties, but it's just easier to understand for later